python/ray/tune/integration/wandb.py

```diff
@@ -326,6 +325,8 @@ def setup(self):
             os.path.expanduser(self.api_key_path) if self.api_key_path else None
         )
         _set_api_key(self.api_key_file, self.api_key)
+        wandb.require("service")
+        wandb.setup(settings=wandb.Settings(start_method="spawn"))
```

Comment:
@krfricke any concern with using spawn here instead of thread? The tests should be catching any problems right?

Reply:
We don't have any proper end to end wandb testing for multiple parallel trials.

@captain-pool did you test these changes with a test run?

Reply:
I ran the pytest runs. I was not aware there were no test for multiple parallel trials. I'll add a test that does the same.